### PR TITLE
iOS now uses `refreshable`

### PR DESCRIPTION
**Pull request by stuartbreckenridge**
AccountManager now has an async function for refreshing accounts (for review)

### CODE REVIEW

**Comment by vincode-io** [comment]
I'm really behind on the WWDC sessions for concurrency, so correct me if I'm wrong. Shouldn't we be throwing something now instead of returning the tuple `AsyncRefreshResult`?

**Comment by stuartbreckenridge** [comment]
I thought about using `withUnsafeThrowingContinuation` but the problem is that a continuation can only run once. 

If you have three accounts and the second of those has an error, you can't throw an error there and continue on to refresh the third, because you've used the continuation already.

**Comment by stuartbreckenridge** [comment]
fb500ce: `refreshAll()` will now throw in situations where there is an connection error. Other refresh errors are not surfaced.

In refreshable it's now `try? await AccountManager.shared.refreshAll()`

**Comment by brentsimmons** [comment]
I recall that NetNewsWire was one of the apps that people were using to work on concurrency — is that fork somewhere around, still? Is it worth looking at? (Maybe we already are?)

**Comment by stuartbreckenridge** [comment]
https://github.com/kavon/NetNewsWire/pull/1

**Comment by stuartbreckenridge** [comment]
I will take a swing at doing this without using continuations.

File: Account/Sources/Account/AccountManager.swift [hunk 1]
```diff
@@ -12,6 +12,9 @@ import RSWeb
 import Articles
 import ArticlesDatabase
 
+public typealias AsyncRefreshResult = (Bool, [Error]?)
+
+
```
Comment (vincode-io) [inline Account/Sources/Account/AccountManager.swift:17]: This can go away now, right?

**Pull request merged**